test/python/framework/materials/xs_combined.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+if "opensn_console" not in globals():
+    from mpi4py import MPI
+    from pyopensn.xs import MultiGroupXS
+else:
+    barrier = MPIBarrier
+
+
+if __name__ == "__main__":
+    xs_1 = MultiGroupXS()
+    xs_1.CreateSimpleOneGroup(sigma_t=1, c=0.5)
+
+    xs_2 = MultiGroupXS()
+    xs_2.CreateSimpleOneGroup(sigma_t=2, c=1./3.)
+
+    xs_combined = MultiGroupXS()
+    combo = [
+        ( xs_1, 0.5 ),
+        ( xs_2, 3.0 )
```

Comment:
@ragusa I am doing this, because it directly maps onto `vector<pair<XS, float>>`, but if you want a dict here `{ "xs" : xs, "factor": <float> }`, I can do that...

Reply:
no. I am good. I was just provided a suggestion when you asked how the user-facing stuff should look like